### PR TITLE
Classification page

### PR DESCRIPTION
**Pull request by gdaswegan**

### CODE REVIEW

**Comment by gdaswegan** [comment]
Don't pull this in yet - gotta modify files so that this doesn't have all of my rest work

**Comment by gdaswegan** [comment]
Okay to pull in - Liang is going to be looking at this further and doing more modifications to make it look nicer and cleaner, but this is a good baseline.

File: dmsite/frontend/src/ClassificationPage/ClassificationPage.jsx [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+    this.toggle = this.toggle.bind(this);
+    this.state = {
+      activeTab: 0,
+      results: [],
+    };
+  }
+
+  toggle(tab) {
+    console.log("switching to " + tab);
```
Comment (lzcheung) [inline dmsite/frontend/src/ClassificationPage/ClassificationPage.jsx]: use console.debug for easier filtering of messages

Comment (gdaswegan) [reply dmsite/frontend/src/ClassificationPage/ClassificationPage.jsx]: Oops, left that in there by accident and forgot to remove it. Thanks for noticing!

**Pull request merged**